src/ccm.rs

```diff
@@ -1,11 +1,14 @@
 //! AES-CCM implementation.
 
-use aes::block_cipher_trait::BlockCipher;
+// Re-export the `aead` crate
+pub use aead;
```

Comment:
Is this really necessary here? We already re-export it in `lib.rs` and I don't notice a difference when I remove this line.

Reply:
Whoops, sorry about that. Removed.